misc/attachments.md

```diff
@@ -4,16 +4,18 @@
 
 ## `Attachment`
 
-Represents a single local file attached to any database object. It may be uploaded by the user,
-received by an e‑mail, or generated by the server.
+Represents a single local file attached to any database object. It may be uploaded by the user, received by an e‑mail, or generated by the server.
```

Comment:
Je nejaky dovod, preco si zrusil zalomanie riadku? Aj source MD by mal mat nejaky limit na dlzku riadku (ak je to technicky mozne) aby sa dobre cital. Idealne rovnaky ako ostatne zdrojaky.